addons/containerd/1.6.15/host-preflight.yaml

```diff
@@ -14,6 +14,18 @@ spec:
           - pass:
               when: "centos = 8"
               message: "containerd addon supports centos 8"
+          - pass:
+              when: "centos = 9"
+              message: "containerd addon supports centos 9"
+          - pass:
+              when: "rhel = 9"
+              message: "containerd addon supports rhel 9"
+          - pass:
+              when: "ol = 9"
+              message: "containerd addon supports ol 9"
+          - pass:
+              when: "rocky = 9"
+              message: "containerd addon supports rocky 9"
```

Comment:
I can't help but wonder why we don't also have `rhel = 8`/`ol = 8` or `rocky = 8` checks

Reply:
added